### PR TITLE
fix sg name

### PR DESCRIPTION
**Pull request by olasaadi99**

### CODE REVIEW

File: pkg/awsvpc/examples/input/input_aws_sg_1.json [hunk 1]
```diff
@@ -1463,8 +1532,9 @@
...
             "OwnerId": "OwnerId:8",
             "State": "available",
-            "Tags": [],
-            "VpcId": "VpcId:43"
+            "Tags": null,
+            "VpcId": "VpcId:40"
         }
-    ]
+    ],
+    "config-file-git-sha1": "$Id$"
```
Comment (haim-kermany) [inline pkg/awsvpc/examples/input/input_aws_sg_1.json:1539]: please notice,
the correct way to get a the config file from the teraform example repository is to it check out from the repository.

File: pkg/awsvpc/parser.go [hunk 2]
```diff
@@ -290,8 +290,11 @@ func (rc *AWSresourcesContainer) getSGconfig(
 		if err != nil {
 			return err
 		}
-		sgName := getResourceName(sg.Tags, *sg.GroupId)
-		commonvpc.NewSGResource(sgName, *sg.GroupId, vpc, NewAWSSGAnalyzer(sg), sgMap, sgLists)
+		sgName := *sg.GroupId
```
Comment (haim-kermany) [inline pkg/awsvpc/parser.go:293]: write a method that get *types.SecurityGroup
and return the sg name.
use it here, and in 
(sga *AWSSGAnalyzer) Name()

**Review by haim-kermany — CHANGES_REQUESTED** [review]

File: pkg/commonvpc/parser.go [hunk 3]
```diff
@@ -133,7 +133,9 @@ func UpdateConfigWithSubnet(name, uid, zone, cidr, vpcUID string, res *vpcmodel.
 	return subnetNode, nil
 }
 
-func NewSGResource(name, uid string, vpc vpcmodel.VPC, analyzer SpecificSGAnalyzer,
+// pairingID is the identifier for the sgs, in aws it is the name of the sg, and in aws it is groupID
+// it is used later for
```
Comment (haim-kermany) [inline pkg/commonvpc/parser.go:137]: please fix the comment:
1. should be ibm, not aws.
2. finish the sentence...

Comment (olasaadi99) [reply pkg/commonvpc/parser.go]: fixed

File: pkg/awsvpc/sg_analysis.go [hunk 4]
```diff
@@ -34,8 +34,16 @@ func NewAWSSGAnalyzer(sg *types.SecurityGroup) *AWSSGAnalyzer {
 	return res
 }
 
+func getSGName(sg *types.SecurityGroup) *string {
+	sgName := sg.GroupId
+	if sg.GroupName != nil && *sg.GroupName != "" {
```
Comment (haim-kermany) [inline pkg/awsvpc/sg_analysis.go:39]: in case there is no sg.GroupName, we should try to take it from the tags

**Review by haim-kermany — APPROVED** [review]

**Review by adisos — APPROVED** [review]

**Pull request merged**